ui/src/components/OverviewTable.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Table, TableHeader, TableBody, sortable, SortByDirection } from '@patternfly/react-table';
+import { Redirect } from 'react-router-dom';
+
+class OverviewTable extends React.Component {
+  constructor (props) {
+    super(props);
+
+    this.state = {
```

Comment:
Will the state management get replaced by Redux once that is introduced?

Reply:
Yes any state that relates to the user data can be moved to redux. I will switch this to using the preloaded state which is populate here: https://github.com/aerogear/mobile-security-service/blob/master/ui/src/index.js#L9 and convert this state management to using redux.